### PR TITLE
[WiP] Add 3-part version of "Meetups in Space" for ActiveRecord practice

### PR DESCRIPTION
**Pull request by cmkoller**
Draft of 3-part challenge for Week 5 (Tues, Weds, Thurs). Each part is intended to focus on the ActiveRecord material presented that day.

Lessons for those three days are:
1. ActiveRecord Migrations
2. ActiveRecord Querying/Associations
3. ActiveRecord Validations

### CODE REVIEW

File: lessons/meetups-in-space-1/meetups-in-space-1.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+When your ER diagram is finished, create the migrations required to set up your database! Make sure to try rolling back each migration (`rake db:rollback`) to make sure that works, before you consider it finished.
+
+When you finish these steps, start setting yourself up for future work on the above user stories. Don't worry yet about retrieving information from the database - that'll happen tomorrow - but you can get ready for that step by setting up the pages you'll need, and your paths in `app.rb`.
+
+### Tips
+
+1. **Before you start writing any code**, take some time to think about what your database should look like and draw an ER diagram.
+2. Work on completing one user story at a time.
+3. You will need to use a [has_many :through association][has-many-through].
```
Comment (radavis) [inline lessons/meetups-in-space-1/meetups-in-space-1.md]: Associations aren't covered in this part of the assignment. I would move this comment to the assignment that deals with associations.

File: lessons/meetups-in-space-2/meetups-in-space-2.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+### Learning Objectives
+
+* Use ActiveRecord to associate tables
+* Retrieve information from a database using ActiveRecord
+
+### Instructions
+
+- Create ActiveRecord models for each table in your database. They should be saved under `app/models`.
+- Set up your models to use ActiveRecord associations. (This may involve making changes to your original plan for your schema - if that is the case, change your schema by adding new migrations, and be sure to update your ER diagram accordingly!)
+- Once both of these steps are done, continue to fulfill the [user stories](meetups-in-space-1) specified yesterday by using ActiveRecord to query your database and displaying that information on your erb pages.
```
Comment (radavis) [inline lessons/meetups-in-space-2/meetups-in-space-2.md]: Just a nitpick, I typically use asterisks for top-level bullets, then alternate asterisks and dashes when indenting.

* item one
  - subitem one
  - subitem two
    * another item
* item two

File: lessons/meetups-in-space-1/meetups-in-space-1.md (hunk 1 above)
Comment (cmkoller) [inline lessons/meetups-in-space-1/meetups-in-space-1.md]: Good point! I ended up just removing the whole tips section here - it seems kind of redundant for the first assignment's material, and it's covered again in the second assignment's material. What do you think?

File: lessons/meetups-in-space-2/meetups-in-space-2.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+### Learning Objectives
```
Comment (radavis) [inline lessons/meetups-in-space-2/meetups-in-space-2.md]: I think that naming these articles after the topic would be helpful: `lessons/meetups-in-space-2-associations`

**Comment by radavis** [comment]
I think that naming the folders after the topic would be helpful: `lessons/meetups-in-space-2-associations`

**Pull request merged**